sos/plugins/__init__.py

```diff
@@ -1081,7 +1097,7 @@ def add_journal(self, units=None, boot=None, since=None, until=None,
         journal_cmd = "journalctl --no-pager "
         unit_opt = " --unit %s"
         boot_opt = " --boot %s"
-        since_opt = " --since %s"
+        since_opt = " --since '%s'"
```

Comment:
FIxes should be in their own patch, not buried in a large feature patch.

Reply:
I'll do that next time I guess?